src/conmon.c

```diff
@@ -22,6 +22,53 @@
 #include <sys/prctl.h>
 #include <sys/stat.h>
 
+static int open_files_max_fd;
```

Comment:
can these additions live in a different file?

Reply:
sure, moved to a new file